src/hooks/hooks.ts

```diff
@@ -1,12 +1,13 @@
 import { BeforeAll, AfterAll, Before, After, Status } from "@cucumber/cucumber";
 import { Browser, BrowserContext, Page, chromium } from "@playwright/test";
 import { pageFixture } from "./pageFixture";
+import { authConfig } from '../../auth.config';
 
 
 let browser: Browser;
 let adminContext: BrowserContext;
-let Context: BrowserContext;
 let Page: Page;
+let adminPage: Page;
 
 BeforeAll(async () => {
```

Comment:
Avoid arrow function !!  Please refactor all of this arrow func to `async function () {}`.
Ref: https://cucumber.io/docs/cucumber/api/?lang=javascript#hooks

Reply:
Done